pkg/buildpack/multi_architecture_helper.go

```diff
@@ -42,9 +42,9 @@ func (m *MultiArchConfig) Targets() []dist.Target {
 // it will return an array with all the platform root folder where the buildpack.toml file were copied to
 func (m *MultiArchConfig) CopyConfigFiles(baseDir string) ([]string, error) {
 	var filesToClean []string
-	for _, target := range m.Targets() {
-		// TODO we are not handling distributions versions yet
```

Comment:
I solved this by adding the `dist.ExpandTargetDistributions` helper method, when we expand the distributions for a given target then we can know which version we are referring to